gnomad/utils/annotations.py

```diff
@@ -1,11 +1,13 @@
 # noqa: D100
 
+import json
 import logging
 from typing import Any, Dict, List, Optional, Set, Tuple, Union
 
 import hail as hl
 
 from gnomad.utils.gen_stats import to_phred
+from gnomad.utils.filtering import get_reference_genome
```

Comment:
fix import order

Reply:
I'm assuming this means moving 'get_reference_genome' before 'to_phred' ? just wanted to clarify

Reply:
yup